build.gradle

```diff
@@ -240,13 +240,14 @@ subprojects {
             targetCompatibility = 1.8
         }
 
-        tasks.withType(AbstractCompile).all {
+        tasks.matching { it instanceof JavaCompile || it instanceof GroovyCompile }.all {
```

Comment:
But below you commented out options.compilerArgs, which seems to contradict this.